src/pages/Lend/LoanChannelCategoryControl.vue

```diff
@@ -290,6 +298,14 @@ export default {
 			}
 			return title;
 		},
+		bundleText() {
+			let text = '';
```

Comment:
can we please add a default here: `Support these three loans with just one click`

Reply:
Ready!